package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@LibertyDSNP/sdk",
+  "name": "@liberty-dsnp/sdk",
```

Comment:
Side note that this might change again as I think it will need to match our npm org which is going to be `unfinishedlabs` not our github org, but we can deal with that when we are publishing later this month.